pkgs/development/coq-modules/mathcomp-analysis/default.nix

```diff
@@ -39,7 +38,7 @@ let
   packages = [ "classical" "analysis" ];
 
   mathcomp_ = package: let
-      classical-deps = [ mathcomp.algebra mathcomp-finmap hierarchy-builder ];
+      classical-deps = [ mathcomp.algebra mathcomp-finmap ];
```

Comment:
@CohenCyril I'm not sure I get that change, it seems to me that old (pre MathComp 2) versions of analysis still depend on hierarchy-builder?

Reply:
The dependency is already [added below](https://github.com/NixOS/nixpkgs/pull/237928/files#diff-0bdd8a94e555cca69d7d4900f59a73697c7b083d531c33e865feefe3a5605b13R84). Adding it unconditionally breaks mathcomp < 0.3.4

Reply:
Indeed, thanks for the explanation.